src/Lecture2.hs

```diff
@@ -70,7 +74,15 @@ return the removed element.
 >>> removeAt 10 [1 .. 5]
 (Nothing,[1,2,3,4,5])
 -}
-removeAt = error "TODO"
+removeAt :: Int -> [b] -> (Maybe b, [b])
+removeAt i [] = (Nothing, [])
```

Comment:
You don't use the `i` variable here so you can replace it with the underscore pattern
```suggestion
removeAt _ [] = (Nothing, [])
```